build.cake

```diff
@@ -61,12 +87,43 @@ Task("Test")
     .IsDependentOn("Build")
     .Does(() =>
     {
-        var projectFiles = GetFiles("./tests/**/*.csproj");
-        foreach(var file in projectFiles)
+        var settings = new DotNetCoreTestSettings {
+			Logger = "trx;logfilename=" + testResultsPath
+		};
+		
```

Comment:
There seems to be something wrong with the indentation. Mixed tabs and spaces?

Reply:
Can the cake file maybe included in the editorconfig? For encoding and tabs/spaces and indentation?